clients/src/main/java/org/apache/kafka/common/record/RecordConversionStats.java

```diff
@@ -16,20 +16,26 @@
  */
 package org.apache.kafka.common.record;
 
-public class RecordsProcessingStats {
+public class RecordConversionStats {
 
-    public static final RecordsProcessingStats EMPTY = new RecordsProcessingStats(0L, 0, -1);
+    public static final RecordConversionStats EMPTY = new RecordConversionStats(0L, 0, -1);
```

Comment:
Not from this patch, but it seems more natural to use 0 for `conversionTimeNanos` here. If we didn't convert anything, than I'd expect it took no time 😉.

Also, maybe we can add a no-arg constructor which initializes everything to 0. We have this in at least one other location.